lib/core.ps1

```diff
@@ -427,8 +428,11 @@ function shim($path, $global, $name, $arg) {
     }
 
     if($path -match '\.(exe|com)$') {
+        if(!(Test-HelperInstalled -Helper Shim)) {
+            scoop install 'main/scoop-shim'
```

Comment:
Manifest is missing binary. Is it intentional?

Reply:
Yes, what would be the purpose of shimming the shim? :smile:

Reply:
You can use it then from anywhere and shim anything. Nice for users who want to shim custom executables.

Small workaround for #3303

Reply:
I would like to avoid recursions :grin:

It could just be copied from `~/scoop/apps/scoop-shim/current/shim.exe`